velox/exec/tests/utils/PlanBuilder.h

```diff
@@ -249,54 +249,17 @@ class PlanBuilder {
   PlanBuilder& optionalFilter(const std::string& optionalFilter);
 
   /// Adds a TableWriteNode to write all input columns into an un-partitioned
-  /// un-bucketed Hive table without collecting statistics without compression.
+  /// un-bucketed Hive table without compression.
   ///
   /// @param outputDirectoryPath Path to a directory to write data to.
   /// @param fileFormat File format to use for the written data.
+  /// @param aggregates Aggregations for column statistics collection during
```

Comment:
Let's document what aggregations are allowed here. Are these min, max, sum and approx_distinct?